src/machine/machine_table.c

```diff
@@ -12079,46 +12077,6 @@ const machine_t machines[] = {
         .snd_device = NULL,
         .net_device = NULL
     },
-    /* Unknown PS/2 KBC. */
```

Comment:
Why are you removing this machine?

Reply:
I did not know that I accidently removed, but I readded that machine. Sorry!